### PR TITLE
Port to Elixir 1.4.2 and latest deps version

### PR DESCRIPTION
**Pull request by freandre**
Hi, 

I've made some modifications to be able  to use your library. Not being an elixir black belt there is maybe some cleaner way to achieve the same things (thinking of the delegate part)

Have a nice day ;)

Fred

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -47,7 +47,7 @@ defmodule Bitmap.Mixfile do
   defp package do
     [
       files: ["lib", "mix.exs", "README*", "LICENSE*"],
-      contributors: ["Kiran Danduprolu", "parroty"],
+      contributors: ["Kiran Danduprolu"],
```
Comment (hashd) [inline mix.exs]: Please add your name as contributor here and keep parroty's name in there too.

**Comment by hashd** [comment]
Thanks for your PR, just raised a single comment against it.

If you can address that, we are good to merge.

**Comment by fredericandre** [comment]
Done, I don't know what I've previously messed up with parroty contributor : /

**Comment by hashd** [comment]
Can you squash these 2 commits to 1?

**Comment by freandre** [comment]
It should be good now

**Comment by Fnux** [comment]
@hashd looks like you forgot this PR ;)

**Comment by hashd** [comment]
@Fnux Thanks for reminding, will work on merging this

**Comment by hashd** [comment]
Merged in #6

**Pull request closed without merging**